charts/s1-agent/values.yaml

```diff
@@ -28,6 +29,7 @@ configuration:
       helper_healthcheck_retry: 60 # amount of tries to verify the helper service is running before starting the agent.
       helper_healthcheck_interval: 5 # time to wait between each try. (seconds)
       fips_enabled: # to enable/disable FIPS mode, set to 'true'/'false'
+      ebpf_enabled: # to disable EBPF mode, set to 'false' (Unsupported in EKS Fargate)
```

Comment:
1. I'm not aware of the full picture, but why can't customers just use PO or something? The agent fails to startup with ebpf_enabled?
2. Don't we have a more "generic way" to pass configurations with helm?

Reply:
1. Right, it is for cases where the agent fails to start with ebpf enabled.
2. There are multiple ways, like using configmap, but this way is fine as well, converting values to environment variables.